akka-docs/src/main/paradox/typed/cluster-sharding.md

```diff
@@ -301,34 +304,78 @@ directly to the `ActorRef`, including messages that the actor sends to itself, a
+
+Or using the `withIdleEntityPassivation` method on `ClusterShardingSettings.PassivationStrategySettings`.
 
-### Least recently used passivation strategy
+If the default strategy is not appropriate for particular workloads and access patterns, a [custom passivation
+strategy](#custom-passivation-strategies) can be created with configurable replacement policies, active entity limits,
+and idle entity timeouts.
 
-The **least recently used** passivation strategy passivates those entities that have the least recent activity when the
-number of active entities passes a specified limit. The configurable limit is for a whole shard region and is divided
-evenly among the active shards in each region. Configure automatic passivation to use the least recently used
-passivation strategy, and set the limit for active entities in a shard region:
+### Custom passivation strategies
 
-@@snip [passivation least recently used](/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala) { #passivation-least-recently-used type=conf }
+To configure a custom passivation strategy, create a configuration section for the strategy under
+`akka.cluster.sharding.passivation` and select this strategy using the `strategy` setting. The strategy needs a
+_replacement policy_ to be chosen, an _active entity limit_ to be set, and can optionally [passivate idle
+entities](#idle-entity-passivation). For example, a custom strategy can be configured to use the [least recently used
+policy](#least-recently-used-policy):
 
-Or enable the least recently used passivation strategy and set the active entity limit using the
-`withLeastRecentlyUsedPassivationStrategy` method on `ClusterShardingSettings`.
+@@snip [custom passivation strategy](/akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingSettingsSpec.scala) { #custom-passivation-strategy type=conf }
 
-#### Segmented least recently used strategy
+The active entity limit and replacement policy can also be configured using the `withPassivationStrategy` method on
+`ClusterShardingSettings`, passing custom `ClusterShardingSettings.PassivationStrategySettings`.
 
-A variation of the least recently used passivation strategy can be enabled that divides the active entity space into
-multiple segments to introduce frequency information into the strategy. Higher-level segments contain entities that
+### Least recently used policy
+
+The **least recently used** policy passivates those entities that have the least recent activity when the number of
+active entities passes the specified limit.
+
+**When to use**: the least recently used policy should be used when access patterns are recency biased, where entities
+that were recently accessed are likely to be accessed again. See the [segmented least recently used
+policy](#segmented-least-recently-used-policy) for a variation that also distinguishes frequency of access.
```

Comment:
👍 , that's exactly the kind of "when to use" that I had in mind. Good to have it.